internal/util/cmdutil/cmdutil.go

```diff
@@ -150,7 +150,18 @@ func WriteFnOutput(dest, content string, fromStdin bool, w io.Writer) error {
 func WriteToOutput(r io.Reader, w io.Writer, outDir string) error {
 	var outputs []kio.Writer
 	if outDir != "" {
-		err := os.MkdirAll(outDir, 0700)
+		_, err := os.Stat(outDir)
+
+		if err == nil || os.IsExist(err) {
```

Comment:
Shouldn't we do this chek in the beginning of the command ? as a pre-run step ?

Reply:
Good idea. Done.